internal/eval/store_eval.go

```diff
@@ -316,3 +317,628 @@ func evalSETEX(args []string, store *dstore.Store) *EvalResponse {
 
 	return evalSET(newArgs, store)
 }
+
+// SETBIT key offset value
+func evalSETBIT(args []string, store *dstore.Store) *EvalResponse {
+	var err error
+
+	if len(args) != 3 {
+		return &EvalResponse{
+			Result: nil,
+			Error:  diceerrors.ErrWrongArgumentCount("SETBIT"),
+		}
+	}
+
+	key := args[0]
+	offset, err := strconv.ParseInt(args[1], 10, 64)
+	if err != nil {
+		return &EvalResponse{
+			Result: nil,
+			Error:  diceerrors.ErrIntegerOutOfRange,
+		}
+	}
+
+	value, err := strconv.ParseBool(args[2])
+	if err != nil {
+		return &EvalResponse{
+			Result: nil,
+			Error:  diceerrors.ErrIntegerOutOfRange,
```

Comment:
Same as above, you can use `diceerrors.ErrGeneral` with the exact error message.